modules/mono/editor/GodotTools/GodotTools/Utils/User32Dll.cs

```diff
@@ -0,0 +1,10 @@
+using System.Runtime.InteropServices;
+
+namespace GodotTools.Utils
+{
+  public static class User32Dll
+  {
+    [DllImport("user32.dll")]
+    public static extern bool AllowSetForegroundWindow(int dwProcessId);
+  }
+}
```

Comment:
Being outside the Rider folder this should use 4 spaces indentation and a new line at the end of the file.